src/screens/Connections/Connections.tsx

```diff
@@ -22,7 +22,11 @@ import EmptyState from './components/EmptyState/EmptyState';
 import Switch from './components/Switch/Switch';
 import {flights} from '../../data';
 
-const Connections = () => {
+type Props = {
+  navigation: any;
```

Comment:
you've defined navigation type. Why use `any`? Btw consider to use `useNavigation` hook instead. Pass navigation type as generic and then all methods will be highlighted